.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,52 @@
+# SPDX-FileCopyrightText: 2021 - 2023 Mewbot Developers <mewbot@quicksilver.london>
+#
+# SPDX-License-Identifier: BSD-2-Clause
+
+name: Publish to PyPI
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  publish:
+    name: Publish Packages
+    runs-on: ubuntu-latest
+
+    if: always()
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Python ${{ env.DEFAULT_PYTHON }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ env.DEFAULT_PYTHON }}
+          cache: pip
+          cache-dependency-path: |
+            requirements.txt
+            requirements-dev.txt
+
+      - name: Install dependencies
+        run: pip install setuptools wheel
+
+      - name: Build distribution packages
+        run: |-
+          rm -Rf build; python setup.py ''     sdist bdist_wheel
+          rm -Rf build; python setup.py 'core' sdist bdist_wheel
+          rm -Rf build; python setup.py 'api'  sdist bdist_wheel
+          rm -Rf build; python setup.py 'io'   sdist bdist_wheel
+
+      - name: "Archive installation artifacts"
+        uses: actions/upload-artifact@v2
+        with:
+          name: builds
+          path: dist/*
+
```

Comment:
It's not perfectly clear to me how the version number for the package is being set before upload?

Reply:
Currently...it's not! It's stuck at 0.0.1 forever right now.